backend/data/scrapers/payload.py

```diff
@@ -26,70 +86,26 @@ def create_payload(size, content_type, year = LIVE_YEAR):
-                            "bool": {
-                                "minimum_should_match": "100%",
-                                "should": [
-                                    {
-                                        "query_string": {
-                                            "fields": [f"{content_type}.implementationYear"],
-                                            "query": f"*{year}*",
-                                        }
-                                    }
-                                ],
-                            }
-                        },
-                        {
-                            "bool": {
-                                "minimum_should_match": "100%",
-                                "should": [
-                                    {
-                                        "query_string": {
-                                            "fields": [f"{content_type}.studyLevelValue"],
-                                            "query": "*ugrd*",
-                                        }
-                                    }
-                                ],
-                            }
-                        },
-                        {
-                            "bool": {
-                                "minimum_should_match": "100%",
-                                "should": [
-                                    {
-                                        "query_string": {
-                                            "fields": [f"{content_type}.active"],
-                                            "query": "*1*",
-                                        }
-                                    }
-                                ],
-                            }
-                        },
-                    ],
-                ],
-                "filter": [{"terms": {"contenttype": [content_type]}}],
-            }
-        },
-        "sort": [{f"{content_type}.code_dotraw": {"order": "asc"}}],
-        "from": 0,
-        "size": size,
-        "track_scores": True,
-        "_source": {
-            "includes": [
-                "*.code",
-                "*.name",
-                "*.award_titles",
-                "*.keywords",
-                "urlmap",
-                "contenttype",
-            ],
-            "excludes": ["", None],
-        },
+            "siteId": "unsw-prod-pres",
+            "contentType": content_type,
+            "queryParams": [{
+                "queryField": "implementationYear",
+                "queryValue": str(year)
+            },
+            # {
+            #     "queryField": "parentAcademicOrg",
+            #     "queryValue": "57a56ceb4f0093004aa6eb4f0310c7ae"
+            # },
+            # {
+            #     "queryField": "studyLevel",
+            #     "queryValue": "undergraduate"
+            # },{
+            #     "queryField": "studyLevelValue",
+            #     "queryValue": "ugrd"
+            # }
```

Comment:
mayb we should uncomment these (instead of filtering) until we stop filtering, for query time. 
NOTE: not sure if this will work for specs and programs though...

Reply:
query time is like <1min in worst case (scraping courses) so not rly a problem